autonomic-administration-algorithms/src/main/java/br/com/autonomiccs/autonomic/administration/algorithms/ClusterAdministrationHeuristicAlgorithm.java

```diff
@@ -85,12 +85,8 @@
     public boolean canPowerOffHost(HostResources hostToPowerOff, CloudResources cloudResources);
 
     /**
-     * Checks if the consolidation agent can disable others idle host in the
-     * given cluster.
-     *
-     * @param cluster
-     *            load (idle resource, total).
-     * @return true if can power off a host.
+     * Checks if the administration agent can disable idle hosts considering the whole environment
```

Comment:
This one I think it would be better:

> This method checks if the administration agent can disable idle hosts given the whole cloud environment current state.